hw/ip/otbn/dv/otbnsim/sim/wsr.py

```diff
@@ -145,7 +145,7 @@ def set_unsigned(self, value: int) -> None:
         the EDN bus and supplies the simulator with an RND value when a fresh
         one is seen on the EDN bus).
         '''
-        assert 0 <= value < (1 << 256)
+        assert 0 <= value < (1 << 256), value
```

Comment:
I'd probably drop this? (I agree that it can't hurt, but I'd generally suggest not leaving the debug prints in)